systemtests/src/test/java/io/enmasse/systemtest/bases/web/WebConsoleTest.java

```diff
@@ -74,6 +74,28 @@ protected void doTestCreateDeleteAddress(Destination... destinations) throws Exc
         assertWaitForValue(0, () -> consoleWebPage.getResultsCount(), new TimeoutBudget(20, TimeUnit.SECONDS));
     }
 
+    protected void doTestCreateDeleteDurableSubscription(Destination... destinations) throws Exception {
+        consoleWebPage = new ConsoleWebPage(selenium, getConsoleRoute(sharedAddressSpace), addressApiClient,
+                sharedAddressSpace, defaultCredentials);
+        consoleWebPage.openWebConsolePage();
+        consoleWebPage.openAddressesPageWebConsole();
+
+        for (Destination dest : destinations) {
+            //create topic
+            consoleWebPage.createAddressWebConsole(dest);
+
+            //create subscription
+            Destination subscription = Destination.subscription(dest.getType() + "-subscriber");
```

Comment:
I think the subscription Destination instance should reference the topic address i.e. dest. As it is, I believe you are creating all the subscriptions against the first topic in the drop down list.